src/keycodes/codes.js

```diff
@@ -10,26 +11,26 @@ export default {
   ctrl: 17,
   alt: 18,
   'pause/break': 19,
-  'caps lock': 20,
+  capslock: 20,
```

Comment:
What's these do for?

Reply:
`caps lock` was wrong. 
https://keycode.info/